### PR TITLE
OLM Definition v4.4.7

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by harshavardhana — DISMISSED** [review]

File: olm.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  --deploy-dir resources/base \
+  --crds-dir resources/base/crds \
+  --manifests \
+  --metadata \
+  --output-dir bundles/$RELEASE \
+  --channels stable
+
+myenv=$EXAMPLE yq -i e '.metadata.annotations.alm-examples |= ("${myenv}" | envsubst)' bundles/$RELEASE/manifests/minio-operator.clusterserviceversion.yaml
+
+miniocontainer="minio/operator:v$RELEASE" yq -i e '.metadata.annotations.containerImage |= env(miniocontainer)' bundles/$RELEASE/manifests/minio-operator.clusterserviceversion.yaml
```
Comment (harshavardhana) [inline olm.sh:18]: for OLM perhaps we can move to `quay.io/minio/operator` as the default image @dvaldivia

File: olm.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+yq eval-all -i '. as $item ireduce ({}; . * $item )' bundles/$RELEASE/manifests/minio-operator.clusterserviceversion.yaml resources/templates/olm-template.yaml
+
+cp bundle.Dockerfile bundles/$RELEASE/
```
Comment (harshavardhana) [inline olm.sh:22]: missing newline

File: resources/templates/olm-template.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
+metadata:
+  annotations:
+    categories: "AI/Machine Learning,   Big Data,  Cloud Provider, Storage"
```
Comment (harshavardhana) [inline resources/templates/olm-template.yaml:3]:
```suggestion
    categories: "AI/Machine Learning, Big Data, Cloud Provider, Storage"
```

File: resources/templates/olm-template.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    ## Prerequisites for enabling this Operator
+
+    * At least Kubernetes 1.18
+
+    * [CSR
+    Capability](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/)
+    must be enabled
+
+    * Locally attached volumes for performance or some CSI to provision block
+    storage to the MinIO pods.
```
Comment (harshavardhana) [inline resources/templates/olm-template.yaml:80]: newline missing

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: olm.sh (hunk 1 above)
Comment (dvaldivia) [reply olm.sh:18]: sure

File: olm.sh (hunk 2 above)
Comment (dvaldivia) [reply olm.sh:22]: done

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**